core/core-model/src/main/java/com/hbc/pms/core/model/enums/AlarmStatusEnum.java

```diff
@@ -0,0 +1,15 @@
+package com.hbc.pms.core.model.enums;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.ToString;
+
+@AllArgsConstructor
+@ToString
+public enum AlarmStatusEnum {

```

Comment:
Avoid using the postfix "Enum" in Java. I don't know the name of this convention; I've not seen the "Enum" postfix in Java yet.